src/popover/popover.styles.tsx

```diff
@@ -29,7 +29,7 @@ const getVisibilityStyle = (visible: boolean) => {
             visibility: visible;
             opacity: 1;
             transition: ${Transition.Base};
-            z-index: 2;
+            z-index: 1000;
```

Comment:
would prefer a smaller value like 50 to avoid inflating z-index numbers

Reply:
sorry i think this change is not required. i have reverted it. will create another PR and use a smaller value if the z-index needs to be updated

Reply:
hmm I did notice that the popover is indeed covered by the navbar in certain cases (when the content is short enough to be rendered above but also tall enough to overlap with the navbar) so this is still good to have

Reply:
ya but i was thinking of lowering the z-index of the navbar instead, currently its at 999 on mylegacy, not sure why its so inflated. im not certain what layer the navbar needs to be at but if it needs to be higher than 1, that means the z-index of the popover still has to be higher than 2... i suppose i will bump the popover z-index to 50 to be safe, thanks!